tests/testerina-integration-test/src/test/resources/project-based-tests/mock-tests/src/Mock/tests/main_test.bal

```diff
@@ -1,73 +1,113 @@
 import ballerina/test;
-import SubtractModule;
-import ballerina/math;
+import ballerina/io;
 
 //
-// TEST CASES
+// MOCK FUNCTION OBJECTS
 //
 
```

Comment:
Can we add a test case to mock io:println since it is used in example tests. We also need to change the example tests as well.